addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -193,8 +176,43 @@ function factory(dependencies) {
          *
          * @param {MouseEvent} ev
          */
-        onClickStartAMeetingButton(ev) {
-            this.createMeetingChannel();
+        async onClickStartAMeetingButton(ev) {
+            const meetingChannel = await this.messaging.models['mail.thread'].createGroupChat({
+                default_display_mode: 'video_full_screen',
+                partners_to: [this.messaging.currentPartner.id],
+            });
+            if (!this.exists()) {
+                return;
+            }
+            await meetingChannel.open();
+            if (!meetingChannel.exists()) {
+                return;
+            }
+            await meetingChannel.toggleCall({ startWithVideo: true });
```

Comment:
Maybe there is a technical limitation for now, but ideally the spec says this should be full screen too (but then we might not be able to display the invite form with current code).

This can be done later if problematic.